hook.php

```diff
@@ -341,12 +341,37 @@ function plugin_fields_addWhere($link, $nott, $itemtype, $ID, $val, $searchtype)
     /** @var \DBmysql $DB */
     global $DB;
 
-    $searchopt = &Search::getOptions($itemtype);
-    $table     = $searchopt[$ID]['table'];
-    $field     = $searchopt[$ID]['field'];
+    $searchopt    = &Search::getOptions($itemtype);
+    $table        = $searchopt[$ID]['table'];
+    $field        = $searchopt[$ID]['field'];
+    $pfields_type = $searchopt[$ID]['pfields_type'];
```

Comment:
```suggestion
    $pfields_type = $searchopt[$ID]['pfields_type'] ?? '';
```
`pfields_type` is added byplugin but is not a regular `key` from glpi core